manifests/journald.pp

```diff
@@ -7,6 +7,19 @@
 
   assert_private()
 
+  if $::systemd::journald_persist_log {
```

Comment:
Please don't add `::` after `$`. That's legacy Puppet 3 syntax and not required anymore:
```suggestion
  if $systemd::journald_persist_log {
```